README.md

```diff
@@ -1,12 +1,16 @@
 # News Site
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The project was about building a classical news  or magazine site. I wanted my site to feature nature and outdoors. I love the scandinavian landscapes and I choose to call the site "A fjord with no name". The posted pictures and texts refer to things to do and to find there outdoors.
 
 ## The problem
 
 Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
 
+I started by making three layouts in figma. As this is totally new for me, it took me lots of hours to do so. I am still uncertain, if it was helpful for me or not, because I still did a lot of designing and changing the styles after I started coding.
```

Comment:
To design can take a lot of your time - try to focus on requirements for the project, my recommendation is  to sketch the main idea on a post it note, and if you find it hard to design here is some instagram konto that is good to follow: [Some inspiration for UX/UI design for you](https://instagram.com/uxswipe?utm_medium=copy_link)
 [Some more tips](https://instagram.com/interfacely?utm_medium=copy_link)

Reply:
Thanks Darya, this is really helpful. I feel the design is the hardest to chew on for me right now